client/transactions/fraud-protection/autocompleter.tsx

```diff
@@ -0,0 +1,72 @@
+/**
+ * External dependencies
+ */
+import * as React from 'react';
+import { __ } from '@wordpress/i18n';
+import { addQueryArgs } from '@wordpress/url';
+import apiFetch from '@wordpress/api-fetch';
+
+/**
+ * Internal dependencies
+ */
+import computeSuggestionMatch from 'utils/compute-suggestion-match';
+
+/**
+ * @typedef {Object} Completer
+ */
+
+interface CompletionOption {
+	key: string;
+	label: string;
+}
+
+const setupAutocompleter = ( status: string ): unknown => ( {
+	name: 'transactions',
+	className: 'woocommerce-search__transactions-result',
+	options( term: string ): Promise< CompletionOption[] > {
+		const query = term ? { search_term: term } : {};
+		return apiFetch( {
+			path: addQueryArgs(
+				'/wc/v3/payments/transactions/fraud-outcomes/search',
+				{
+					status,
+					...query,
+				}
+			),
+		} );
+	},
+	isDebounced: true,
+	getOptionIdentifier( option: CompletionOption ): string {
+		return option.label;
+	},
+	getOptionKeywords( option: CompletionOption ): string[] {
+		return [ option.label ];
+	},
+	getOptionLabel( option: CompletionOption, query: string ): JSX.Element {
+		const match = computeSuggestionMatch( option.label, query );
+
+		return (
```

Comment:
I assume if there's no match, it'll display an empty element, right? How about we return if match exists? Like `return match ? ( ..span ) : null` ?

Reply:
I tried this approach, but the behavior didn't change. It's still rendering an empty item. I think the issue is that the `getOptionLabel` only formats the text. ([Reference](https://github.com/woocommerce/woocommerce/blob/trunk/packages/js/components/src/search/index.tsx#L234))

<img width="1172" alt="image" src="https://user-images.githubusercontent.com/16882226/226966876-d1710f7a-b2b2-42f2-a3f1-97457fe3e4d8.png">

Reply:
Added on [626427e](https://github.com/Automattic/woocommerce-payments/pull/5487/commits/626427e681b18fee2e47a87bf8d00e69812dab44).